include/ghex/transport_layer/ucx/endpoint.hpp

```diff
@@ -64,7 +64,7 @@ namespace gridtools {
                     void destroy()
                     {
                         if (!m_moved){
-                            ucs_status_ptr_t ret = ucp_ep_close_nb(m_ep, UCP_EP_CLOSE_MODE_FLUSH);
+                            ucs_status_ptr_t ret = ucp_ep_close_nb(m_ep, UCP_EP_CLOSE_MODE_FORCE);
```

Comment:
I thought I've fixed this? please merge with master

Reply:
I am merged with master. No idea why this is here. Is your other changes (the barrier) present?

Reply:
the barrier is there, very strange. Could you still change it back to flush?

Reply:
This is very strange. It is the only line, which did not get merged. All your other changes from f237b72741b20efad84e18720b65c5ea7f53d5e9 are in. Also, blame shows that my version is much older. So I do not understand why it wasn't merged.

Reply:
This is very scary. Do we have to understand this? What else is missing?